### ISSUE TITLE
isSupported returns true when redirecting stdout on windows

### ISSUE THREAD

**Issue by mdonnalley** [webdiscus/ansis#22]
## Current behaviour

On windows, `ansis.isSupported()` returns `true` when redirecting stdout to a file. On mac (and I'm assuming linux) it returns `false`.

This leads to ansi characters showing up in the file containing the redirected stdout. For example:

```
[1mhello world[22m
```

## Expected behaviour

I expect `ansis.isSupported()` to return `false` when redirecting stdout to a file, regardless of operating system. This would be consistent with the supports-color package

### Reproduction Example

`git clone https://github.com/mdonnalley/ansis-bug.git`
`cd ansis-bug`
`npm install`
`node index.js > output.txt`

On windows:
```
PS C:\Users\username\code\ansis-bug> node index.js > output.txt
PS C:\Users\username\code\ansis-bug> cat output.txt
ansis.isSupported true
supportsColor.stdout false
```

On mac:
```
❯ node index.js > output.txt
❯ cat output.txt
ansis.isSupported false
supportsColor.stdout false
```

## Environment

- OS: Windows
- version of Node.js: 20.13.1
- version of Webpack: NA
- version of the Plugin: NA

## Additional context

NA

**Comment by webdiscus** [comment]
Hello @mdonnalley,

Thanks for the issue report. I will fix it.
What is Windows version: 10, 11, ... ?

**Comment by mdonnalley** [comment]
@webdiscus thanks! I'm on Windows 11

**Comment by webdiscus** [comment]
@mdonnalley 

the issue is fixed in the v3.3.2.
Can you please test it?

**Comment by mdonnalley** [comment]
@webdiscus just tested and it's working as expected - thanks so much for quick turnaround!
